automation_tools/__init__.py

```diff
@@ -1891,6 +1896,9 @@ def downstream_install(admin_password=None, run_katello_installer=True):
         remote_path='/etc/yum.repos.d/satellite.repo')
     satellite_repo.close()
 
+    maintain_url = os.environ.get('MAINTAIN_BASE_URL')
```

Comment:
duplicate code. already assigned maintain_url at 1884 line

Reply:
removed this